src/main/groovy/com/wikia/gradle/marathon/MarathonTask.groovy

```diff
@@ -47,6 +49,12 @@ class MarathonTask extends DefaultTask {
             app.setHealthChecks(healthChecks)
         }
 
+        List<List<String>> constraints = this.stage.resolve(Constraints).getConstraints()
```

Comment:
Does this work even if the app doesn't specify a `constraints` field, or will this somehow cause an NPE?